frontend/src/App.js

```diff
@@ -1,15 +1,26 @@
-
 import './App.css';
+import React, { useState } from 'react';
+import Navigation from './Navigation';
+import StepTime from './StepTime';
+import Chart from './Chart';
+import Graph from './Graph';
+import useStepTime from './useStepTime';
 
 function App() {
+  const [currentView, setCurrentView] = useState('step_time');
+  const stepTime = useStepTime();
+
+  const views = {
+    step_time: <StepTime stepTime={stepTime} />,
+    chart: <Chart stepTime={stepTime} />,
+    graph: <Graph stepTime={stepTime} />,
```

Comment:
Consider more descriptive variable names for things like this. In the future, we may have charts for step time, target zone, overall patient performance, reading this code then wouldn't say much about which of those charts/graph is rendered here.

Consider names like `StepTimeDigits`, `StepTimeSecs`, `StepTimeChart`, just something that immediately sets the reader's expectation